TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/ParameterNode.java

```diff
@@ -172,20 +169,20 @@ private void callDynamicReconfigure(String paramName, boolean paramValue) {
             serviceClient.call(srv_req, new ServiceResponseListener<ReconfigureResponse>() {
                 @Override
                 public void onSuccess(ReconfigureResponse reconfigureResponse) {
-                    Log.i(TAG, "Dynamic Reconfigure success");
+                    mConnectedNode.getLog().info("Dynamic Reconfigure success");
```

Comment:
Could we store the log at the beginning of the function and then reuse this, e.g. add  `Log log = mConnectedNode.getLog();` somewhere at L159